packages/components/src/components/date-picker/date-picker-input.jsx

```diff
@@ -24,6 +24,7 @@ const DatePickerInput = ({
     readOnly,
     value,
     required,
+    ...common_props
```

Comment:
```suggestion
    ...props
```
Just a small nit since we usually forward props using the `..props`